### ISSUE TITLE
type import produces false positive under lodash/import-scope

### ISSUE THREAD

**Issue by AprilArcus** [wix/eslint-plugin-lodash#337]
with `"lodash/import-scope": ["error", "method"]`, named typescript type imports from lodash produce false positive errors:

```
import type { DebouncedFunc } from 'lodash' // error
```

**Comment by jlaustill** [comment]
We are also experiencing this, seems like this library isn't typescript aware, which makes sense because it's written in vanilla js from what I can tell.
